src/store/server/spoolman/actions.ts

```diff
@@ -15,6 +27,7 @@ export const actions: ActionTree<ServerSpoolmanState, RootState> = {
             {
                 request_method: 'GET',
                 path: '/v1/info',
+                use_v2_response: true,
```

Comment:
this and the following uses of `use_v2_response` require a minimum moonraker right?

Reply:
this is backward compatible. if Moonraker is older, this value will be ignored and the answer will also be converted.